3-platform/warehouse/app/pom.xml

```
@@ -9,12 +9,12 @@
         <version>0.0.1-SNAPSHOT</version>
     </parent>
 
-    <artifactId>warehouse-app</artifactId>
+    <artifactId>app</artifactId>
```

dont need to revert this!! 
We have already passed why. 
```suggestion
    <artifactId>app</artifactId>
```